app/views/expenses/_expense.html.erb

```diff
@@ -0,0 +1,16 @@
+<div id="<%= dom_id expense %>" class="expense">
+  <div>
+    <p>
+      <%= link_to expense.name, budget_category_expense_path(@budget_category, expense) %>
+    </p>
+    <p class="date">
+      <i class="fa-regular fa-calendar"></i>
+      <%= expense.time_created %>
+    </p>
+  </div>
```

Comment:
- [ ] I am not able to see the name of the transactions in the UI. Please fix that.


![Screenshot from 2023-03-12 00-31-57](https://user-images.githubusercontent.com/26767383/224507149-db085073-354b-46ca-9ff6-565e5a18e098.png)

Reply:
hi @Deepakdanger thank you for your review